ruby/README.md

```diff
@@ -0,0 +1,5 @@
+# How to run
+
+1. Set api url and api token in app.rb
+2. `bundle install`
+3. `bundle exec ruby app.rb`
```

Comment:
Something something swap out the API key?

Reply:
read step 1 one more time :D

Reply:
I... I'm going to go be dumb somewhere else